### PR TITLE
feat: content changes

### PR DESCRIPTION
**Pull request by lchlipala**
## Description

Related issue: [BL-16855](https://dvsa.atlassian.net/browse/BL-16855)

New notice:
image

Reason For Rejection Updates:
image

Light Vehicle Support:

image

HGV Support Section:
image

## Before submitting (or marking as "ready for review")

- [x] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [x] Have you performed a self-review of the code
- [ ] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

**Review by alexisc-kainos — APPROVED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -33,10 +33,12 @@ <h1 class="heading-xlarge u-space-b15">MOT history API</h1>
 
     <div class="grid-row u-space-t30">
         <div class="column-two-thirds">
-            <p>It's no longer possible to register for this version of the MOT history API.</p>
+            <p>A new version of the MOT history API is available. <a
+                    href="https://documentation.history.mot.api.gov.uk/" target="_blank">You can register to use the new API now.</a></p>
```
Comment (adambrennankainos) [inline docs/index.html]: "You can" should not be part of the link text according to content in the ticket

Comment (lchlipala) [reply docs/index.html]: Good spot @adambrennankainos, thanks. Just changed it. Are you we good to resolve thread now?

**Review by adambrennankainos — APPROVED** [review]

**Review by ccurran01 — APPROVED** [review]

File: docs/index.html [hunk 2]
```diff
@@ -33,10 +33,12 @@ <h1 class="heading-xlarge u-space-b15">MOT history API</h1>
 
     <div class="grid-row u-space-t30">
         <div class="column-two-thirds">
-            <p>It's no longer possible to register for this version of the MOT history API.</p>
+            <p>A new version of the MOT history API is available. You can <a
+                    href="https://documentation.history.mot.api.gov.uk/" target="_blank">register to use the new API now</a>.</p>
         </div>
         <div class="column-two-thirds">
-            <p>Registration for a new MOT history API will be opening shortly.</p>
+            <p> The old API will be turned off later this year. We’ll contact users before this happens. If you are using the old API, you’ll need to <a
```
Comment (sdh100shaun) [inline docs/index.html:40]: Note GDS advice on opening in new window - https://design-system.service.gov.uk/styles/links/

Comment (ccurran01) [reply docs/index.html:40]: Thanks @sdh100shaun. I've updated each of the links on this page to contain `(opens in new tab)` as recommended here 👍

**Pull request merged**